resources/views/auth/register.blade.php

```diff
@@ -73,5 +74,85 @@
             </div>
         </div>
     </div>
+</div-->
+<!-- endsection -->
+
+<div class="container" id="reg-container">
+
+    <h2 id="reg-h2">We are happy to have you here! Please fill out the following lines:</h2>
+
+    <!-- Form -->
```

Comment:
I'm all for this during development but please remove all of these when you're finalizing stuff.

Reply:
Do you mean the comment or the H2?